baselines/Gmail Minimum Viable Secure Configuration Baseline v0.2.md

```diff
@@ -1152,3 +1153,76 @@ To configure the settings for Objectionable content:
 
 #### GWS.GMAIL.18.3v0.2 Instructions
 1.  There is no implementation steps for this policy.
+
+
+## 19. Spam Filtering
+
+This section covers the settings relating to bypassing spam filters.
+
+### Policies
+
+#### GWS.GMAIL.19.1v0.1
+Domains SHALL NOT be added to lists that bypass spam filters.
+
+- _Rationale:_ Allowing an entire domain to bypass the spam filters allows for the potential for a spoofed email within the domain to bypass the filter. Only allowing specific users to bypass helps mitigate the risk.
+- _Last modified:_ April 10, 2024
+- _Note:_ Allowed senders MAY be added.
+
+- MITRE ATT&CK TTP Mapping
+  - [T1566: Phishing](https://attack.mitre.org/techniques/T1566/)
+    - [T1566:001: Phishing: Spearphishing Attachment](https://attack.mitre.org/techniques/T1566/001/)
+    - [T1566:002: Phishing: Spearphishing Link](https://attack.mitre.org/techniques/T1566/002/)
+  - [T1534: Internal Spearphishing](https://attack.mitre.org/techniques/T1534/)
+
+#### GWS.GMAIL.19.2v0.1
+Domains SHALL NOT be added to lists that bypass spam filters and hide warnings.
+
+- _Rationale:_ Allowing an entire domain to bypass the spam filters and hide warnings allows for the potential for a spoofed email within the domain to bypass the filter and prevents the user from knowing. Not adding domains and users helps mitigate the risk.
+- _Last modified:_ April 10, 2024
+
+- MITRE ATT&CK TTP Mapping
+  - [T1566: Phishing](https://attack.mitre.org/techniques/T1566/)
+    - [T1566:001: Phishing: Spearphishing Attachment](https://attack.mitre.org/techniques/T1566/001/)
+    - [T1566:002: Phishing: Spearphishing Link](https://attack.mitre.org/techniques/T1566/002/)
+  - [T1534: Internal Spearphishing](https://attack.mitre.org/techniques/T1534/)
+
+#### GWS.GMAIL.19.3v0.1
+Bypass spam filters and hide warnings for all messages from internal and external senders SHALL NOT be enabled.
+
+- _Rationale:_ Bypassing spam filters and hiding warning for all messages from internal and external senders creates a security risk because of the potential for a malicious message being able to bypass filters. Disabling this feature mitigates the risk.
```

Comment:
Recommend tweaking this rationale because this setting effectively disables spam filtering, creating more than the potential for messages to bypass filters.
```suggestion
- _Rationale:_ Bypassing spam filters and hiding warning for all messages from internal and external senders creates a security risk because all messages are allowed to bypass filters. Disabling this feature mitigates the risk.
```